storage_test.go

```diff
@@ -245,22 +241,18 @@ func TestStorageSnapshots_ListStorageSnapshots(t *testing.T) {
 
 	expected := []Snapshot{
 		{
-			Region:        &Region{Slug: "nyc3"},
+			Regions:       []string{"nyc3"},
 			ID:            "80d414c6-295e-4e3a-ac58-eb9456c1e1d1",
-			VolumeID:      "98d414c6-295e-4e3a-ac58-eb9456c1e1d1",
 			Name:          "my snapshot",
-			Description:   "my description",
 			SizeGigaBytes: 100,
-			CreatedAt:     time.Date(2002, 10, 02, 15, 00, 00, 50000000, time.UTC),
+			Created:       "2002-10-02T15:00:00.05Z",
```

Comment:
any reason to use `string` instead of `time.Time`? Usually typed things > string things.